docs/examples/transformations.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Transformations
+parent: Examples
+nav_order: 4
+---
+# Transformations
+
+With transformations we can manipulate data prior to writing to the data store.
+
+## Aggregate
+With aggregates we can take multiple records and output a single record.
+
+```csharp
+using (var writer = new BulkWriter<int>(connectionString))
+{
+    var items = Enumerable.Range(1, 1000).Select(i => new MyClass { Id = i, Name = "Bob" });
```

Comment:
Minor thing: we wouldn't aggregate an Id - let's rename to something else that *would* make sense as an aggregate.